ui/app/components/transit-key-action/export.js

```diff
@@ -0,0 +1,5 @@
+import Component from '@glimmer/component';
+export default class ExportComponent extends Component {
+  wrapTTL = null;
```

Comment:
Do these need tracked attribute?

Reply:
Yes, i believe so since we're mutating on the template